code/modules/mob/living/carbon/human/species.dm

```diff
@@ -1253,6 +1253,16 @@ GLOBAL_LIST_EMPTY(roundstart_races)
 			target.forcesay(GLOB.hit_appends)
 
 /datum/species/proc/disarm(mob/living/carbon/human/user, mob/living/carbon/human/target, datum/martial_art/attacker_style)
+	if(user.zone_selected == "mouth" && istype(target.wear_mask, /obj/item/clothing/mask/cigarette))
```

Comment:
This is snowflake af. It's something specific to cigarettes, so it should be on the cigarette path

Reply:
the only way i can think of doing that is adding code to the cigarette's process() to check if its user has been disarmed by someone targeting the mouth. is that a good solution?